coordinator/src/node.rs

```diff
@@ -66,7 +66,7 @@ impl Node {
         let margin_trader = margin_trader(trade_params);
         let margin_coordinator = margin_coordinator(trade_params);
 
-        let tomorrow = OffsetDateTime::now_utc().date() + Duration::days(1);
+        let tomorrow = OffsetDateTime::now_utc().date() + Duration::days(2);
```

Comment:
Why +2 days?

Reply:
so it will be open for at least 24h, up to 47 hours and 59 mins.